data/assets/scene/solarsystem/planets/earth/satellites/misc/spacestations.asset

```diff
@@ -3,8 +3,21 @@ local shared = asset.require("util/tle_helper")
 local group = {
   Title = "SpaceStations",
   Url = "http://celestrak.com/NORAD/elements/stations.txt",
-  TrailColor = { 0.9, 0.1, 0.0 }
+  TrailColor = { 0.9, 0.1, 0.0 },
+  Description = [[A collection of space stations (Including the ISS and China’s 
+    Tiangong), along with certian cubesats and satellite constellations from space 
```

Comment:
`certian` -> `certain`

and nitpicky, but would spell including with small "I" on the previous line. Better flow in text